### ISSUE TITLE
Dashboard improvement ideas

### ISSUE THREAD

**Issue by angerson** [tensorflow/build#142]
- [ ] Highlight buttons when hovered, instead of disappearing
- [ ] Indicate when a link or result is internal-only
- [ ] Show pending Kokoro jobs (may not be possible)
- [ ] Show Nightly jobs
- [ ] Adjust styling for favorite-star, remove it, or add new "favorites" section (unsure if still useful, but definitely ugly)
- [ ] Remove extra "piperorigin-revid" section if it is parsed successfully
- [ ] Align CI results on commit view
- [ ] Add JS-based "time since last update" counter in header
- [ ] Rewrite the help section
- [ ] Do not highlight date buttons, it doesn't do anything
- [ ] Indicate difference between GA jobs and non-GA jobs
- [ ] Colorize infrastructure errors differently than regular failures
- [ ] Reconsider card ordering within sections (by number of results?)
- [ ] Improve color contrast on titles
- [ ] Add link to issue tracker or discussion thread on this page
- [ ] "In progress" animation may jump when resetting; unsure
- [ ] Normalize rules for naming jobs (Should be Topic - Details)
- [ ] Dots are currently grouped by GMT instead of PDT (i.e. can see a status dot from "tomorrow")

**Comment by learning-to-play** [comment]
Showing if a job is currently pending / running will be helpful.

**Comment by angerson** [comment]
> Showing if a job is currently pending / running will be helpful.

Yes, it would, but it is not possible because Kokoro does not report pending statuses, as I already mentioned in the Help page and in the design doc I shared internally. I filed a feature request bug with them which is already linked from the design doc.
